React/Views/RCTView.m

```diff
@@ -341,6 +361,136 @@ - (NSString *)accessibilityValue
   }
   return nil;
 }
+#else // [TODO(macOS ISS#2323203)
+- (id)accessibilityValue {
+  if (self.accessibilityRole == NSAccessibilityCheckBoxRole) {
+    for (NSString *state in self.accessibilityState) {
+      id val = self.accessibilityState[state];
+      if (!val) {
```

Comment:
We should avoid implicit boolean conversions https://github.com/microsoft/objc-guide/blob/7bc41b8608d55a696a5b1c44d05be0e7b2496954/MethodsAndImplementations/ImplicitBooleanConversion.md

Reply:
Ohhh, a objc guide ✨